jupyterlab_server/translation_utils.py

```diff
@@ -14,9 +14,13 @@
 from typing import Dict, Pattern, Tuple
 
 import babel
-import entrypoints
 from packaging.version import parse as parse_version
 
+if sys.version_info < (3, 10):
```

Comment:
Might even be worth linking to that thing about why this complexity is required.

Reply:
Thanks